ci/ci_steps.sh

```diff
@@ -29,10 +29,12 @@ then
   exit 0
 elif [ "$TEST_TYPE" == "build_image" ]
 then
-  # The script builds lyft/envoy-build and pushes that image when ci/build_container
+  # The script builds lyft/envoy-build{-centos} and pushes that image when ci/build_container
   # has changed on a push to master.
   echo "lyft/envoy-build pushing..."
   ./ci/build_container/docker_push.sh
+  echo "lyft/envoy-build-centos pushing..."
+  ./ci/build_container_centos/docker_push.sh
```

Comment:
We don't have the CI resources right now to build CentOS. We are working on improving the situation but for now this will need to be disabled by default.

Reply:
commented below.